tensorwaves/data/tf_phasespace.py

```diff
@@ -0,0 +1,58 @@
+"""Phase space generation using tensorflow."""
+
+import numpy as np
+
+import phasespace
+
+import tensorflow as tf
+
+from tensorwaves.interfaces import (
+    PhaseSpaceGenerator,
+    UniformRealNumberGenerator,
+)
+
+
+class TFPhaseSpaceGenerator(PhaseSpaceGenerator):
+    """Implements a phase space generator using tensorflow."""
+
+    def __init__(
+        self, initial_state_mass: list, final_state_masses: list
+    ) -> None:
+        print(phasespace.__file__)
+        self.phsp_gen = phasespace.nbody_decay(
+            initial_state_mass, final_state_masses
+        )
+
+    def generate(
+        self, size: int, random_generator: UniformRealNumberGenerator
```

Comment:
This is what https://github.com/ComPWA/tensorwaves/issues/59 is about, right?

Reply:
yes